lib/index.ts

```diff
@@ -396,6 +428,11 @@ export class UsbbootScanner extends EventEmitter {
 		super();
 		this.boundAttachDevice = this.attachDevice.bind(this);
 		this.boundDetachDevice = this.detachDevice.bind(this);
+
+		// This is an undocumented property
+		if ((<any>usb).INIT_ERROR as any) {
```

Comment:
@zvin @thundron Is there a better way to make TSLint happy here? `INIT_ERROR` is not on the `usb` types we're pulling in